src/main/java/org/tron/common/runtime/RuntimeImpl.java

```diff
@@ -346,20 +403,29 @@ private void create()
 
       AccountCapsule creator = this.deposit
           .getAccount(newSmartContract.getOriginAddress().toByteArray());
+
+      if (RuntimeImpl.isNewVersion()) {  // TODO according to version
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1670&issues=AWbSdHUuSgN4uCclEWnK&open=AWbSdHUuSgN4uCclEWnK)

[ISSUE_KEY]: AWbSdHUuSgN4uCclEWnK